src/gradio_ui.py

```diff
@@ -126,9 +136,12 @@ def pairwise(iterable):
                     # Grab char name from the model history
                     char_name = bot_turn.split(":")[0]
                 # Format the user and bot utterances
-                #pdb.set_trace()
                 user_turn = human_turn.replace("You: ", "")
                 bot_turn = bot_turn.replace(f"{char_name}:", f"**{char_name}:**")
+
+                # Replace [NAME_IN_MESSAGE_REDACTED] with the user's name
+                user_turn = user_turn.replace("[NAME_IN_MESSAGE_REDACTED]", user_name)
+                bot_turn = bot_turn.replace("[NAME_IN_MESSAGE_REDACTED]", user_name)
```

Comment:
Somebody released a script on /g/ which tries to convert CAI dump logs to Pygmalion character settings and chats. However, because of the anonymization, the chats may contain "[NAME_IN_MESSAGE_REDACTED]" in them, which you obviously wouldn't want to have in the conversation history. These lines were meant to help out those people who converted the CAI dumps to Pygmalion conversation history automatically via. that script.

I plan to eventually integrate the entire script into there, but for now, I figure that this would be a nice aid for those using that script so they don't have to manually replace "[NAME_IN_MESSAGE_REDACTED]" with whatever they want to be called in the file. I have never seen the model nor any reports of the model outputting [NAME_IN_MESSAGE_REDACTED].

Reply:
Will do.

Reply:
Thanks, LGTM!